opencti-platform/opencti-front/src/schema/relay.schema.graphql

```diff
@@ -1099,7 +1099,7 @@ input QueryTaskAddInput {
 
 input RetentionRuleAddInput {
   name: String!
-  filters: String!
```

Comment:
100% sure it defaults to an empty filter group ? :)

Reply:
yes if you create a retention rule in the UI, no if you create it via the API...